app/uk/gov/hmrc/customsservicestatusfrontend/connectors/PlannedWorkConnector.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customsservicestatusfrontend.connectors
+
+import uk.gov.hmrc.customsservicestatusfrontend.models.PlannedWork
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.{HeaderCarrier, StringContextOps}
+import uk.gov.hmrc.http.HttpReads.Implicits.*
+
+import javax.inject.{Inject, Named, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class PlannedWorkConnector @Inject() (
+  httpClient:                                                    HttpClientV2,
+  @Named("customsServiceStatusUrl") customsServiceStatusBaseUrl: String
+)(implicit ec: ExecutionContext) {
+
+  private val baseUrl: String = s"$customsServiceStatusBaseUrl/customs-service-status/services/planned-work"
```

Comment:
It's a bit misleading to call this the "baseUrl" when it is the url of a particular endpoint. 

Also, appreciate this may not always be done in other services, but ideally the main path for the service being called would also be obtained from the service configuration, and I don't see why the endpoint specific part cannot also be configurable (would allow the address to change in the underlying service without a code change here).